### ISSUE TITLE
Analytical support training: Siriwan

### ISSUE THREAD

**Issue by aluczko** [aluczko/analytical-projects-AEL#8]
Hi Erik,

Please see below agenda for Siriwan.

I would like for her to work with you per your schedule to cover the following:
Understanding of all BI reports for Quality.
Understanding of the Grader Quality report in Quest.
Understanding of bulletins and how to look for trends.

**Comment by aluczko** [comment]
Reply sent on 11/15/2016

Hi Forum,
During the first meeting, we went over all BI reports for Quality – Siriwan was exceptionally at ease with them and knew what information they encompass.  Same with the Grader Quality in Quest (I clarified the ‘Majors per 1000’ only).  During the remaining sessions I explained in depth the bulletin process, demonstrated how useful GitHub is in managing the bulletins.  She had no problems understating the Nelson rules (I pointed her to the available resources, i.e. the Tutorial available in BI, and external links), and spotting trends on the control charts.  She asked many excellent questions, for instance how is the mean recalculated (I explained it to her by going to TNM in Quest and showing her the steps).
Since we were about to issue a bulletin for GCs, polish, in BKK, I asked her to ‘investigate’ the cause.  She used BI to gather the information, and prepared a thorough response pointing accurately to the product mix as a cause.
Soon after, I prepared a detailed dataset for her (similar to what I obtain for myself and/or Claudia when preparing bulletins) and asked to analyze it on a ‘sub-grade’ level.   She used Excel to process the data.  Her response is in the attachment.  
She was quick to point to possible causes even before looking at the data, a good indicator to me that the knows exactly what product mix the BKK lab is seeing, how is could potentially affect the defect rates; she knew the ‘weaker’/newer graders in the lab, and understood how, for example, unevenness of the GC distribution could impact the lab defect rate.
To sum up her analytical skills: she is one of a very few that I know who can quickly and accurately connect the ‘dots’, draw conclusion from the resources available (BI) and relate them to what she observes in the lab.
Aside from that, remarkable personality, sincere, polite, it was a pleasure meeting with her and discussing such broad range of topics.
What might help her in her further development is probably acquiring more advanced Excel skills.
Thank you,
Erik
